proposals/lip-0048.md

```diff
@@ -25,15 +25,57 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
-### Fee Token ID
+Fees for blockchain transactions are necessary for a variety of reasons, including state growth, node computation, and block space utilization.
 
-Each chain can configure the token used to pay fees. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+The current implementation lacks an automated method for calculating fees in all of these cases. Instead, we automate the calculation of block space fees using a *minimum fee* approach, similar to Bitcoin's mechanism, and allow developers discretion to use the [`payFee`](#payfee) method for fee assignment in other situations, e.g., when transaction implies state growth or more significant computational effort required by a node.
 
-### Minimum Fee per Transaction
+### Transaction Fees
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when the minimum fee per byte is set to zero.
+Each transaction in the Lisk ecosystem has a *transaction fee* associated with it, as defined in [LIP 0068][lip-0068]. Transaction fees prevent spamming a blockchain with unlimited transactions at zero cost. In addition, a part of the transaction fee is provided as a reward to the validators who operate the network and maintain its security, as we explain in detail below. 
```

Comment:
```suggestion
Each transaction in the Lisk ecosystem has a *transaction fee* associated with it, the `fee` property in the transaction schema in [LIP 0068][lip-0068]. Transaction fees prevent spamming a blockchain with unlimited transactions at zero cost. Part of the transaction fee is burned or transferred to a common pool. The remaining part is given as a reward to the validator who generated the block that included the transaction. 
```

Reply:
Also, I would move this paragraph to the very top and remove this section.

Reply:
Agree, this should be merged with the very first paragraph of the rationale.

Reply:
Not sure we should add the burn/pool topic here. The main point of this paragraph is to explain *why* we need fees, and one of the reasons is that rewards validators. I would then keep it as is and just merge it with the very first sentence. Meaning, we just motivate the use of transaction fees.

Reply:
I went for the Greg's suggestion, maybe we don't need to mention the burning here, since this is moved to the Rationale introduction.